script/StaticLint-juia-server.jl

```diff
@@ -0,0 +1,17 @@
+cd(@__DIR__)
+
+using Pkg;
+Pkg.activate(joinpath(pwd(),"StaticLintAtom"))
+Pkg.instantiate()
+
+# using StaticLintAtom
+# include(joinpath(pwd(),"StaticLintAtom","src","StaticLintAtom.jl"))
+
+using LanguageServer
+using SymbolServer
+
+# named_pipe = ARGS[1] # julia.exe path
```

Comment:
```suggestion
```

Reply:
This shouldn't be removed. We need to correct it eventually.